doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -285,6 +285,9 @@ nRF Desktop
   * Manifest semantic version information to the firmware information response in :ref:`nrf_desktop_dfu` when the module is used for SUIT DFU and the SDFW supports semantic versioning (requires v0.6.2 and higher).
   * A missing DTS node compatible with ``zephyr,hid-device`` to the nRF52840 DK in the MCUboot QSPI configuration.
     This ensures support for HID over USB when the USB next stack is selected.
+  * The USB next stack (:ref:`CONFIG_DESKTOP_USB_STACK_NEXT <config_desktop_app_options>`) implies partial erase feature of the nRF SoC FLASH driver (:kconfig:option:`CONFIG_SOC_FLASH_NRF_PARTIAL_ERASE`).
```

Comment:
I would say:

Updated:

- the integration of the USB next stack to imply partial...

Reply:
I was wondering about it. Depends from perspective. Either you added a new imply or updated a Kconfig option to imply something.

Reply:
If you prefer to keep it under updated, I can move it